src/ui/generic-components/TrafficLightButton.swift

```diff
@@ -117,40 +117,55 @@ class TrafficLightButton: NSButton {
     }
 
     private func drawSymbol(_ lineColor: NSColor) {
+        let symbol = NSBezierPath()
         if (type == .fullscreen) {
-            let symbol = NSBezierPath()
-            symbol.move(to: NSMakePoint(bounds.width * 0.25, bounds.height * 0.75))
-            symbol.line(to: NSMakePoint(bounds.width * 0.25, bounds.height * 1 / 3))
-            symbol.line(to: NSMakePoint(bounds.width * 2 / 3, bounds.height * 0.75))
+            var firstPointA, firstPointB, firstPointC, secondPointA, secondPointB, secondPointC: NSPoint!
+            if window_?.isFullscreen ?? true {
```

Comment:
I think it may be safer to default to the fullscreen symbol if we lost the `window_` reference here.

```swift
window_?.isFullscreen ?? false
```